submissions/lisnyak-o/popup-kottans/css/style.css

```diff
@@ -0,0 +1,162 @@
+body {
+  font-family: "Fira Sans", sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  background-color: #fdfdfd;
+  color: rgb(138, 138, 138);
+  text-transform: uppercase;
+}
+.container {
+  max-width: 1230px;
+  margin: 0 auto;
+  padding: 0 15px;
+}
+.menu__wrapper {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+.menu {
+  margin-top: 30px;
+}
+.menu li + li {
+  padding-left: 10px;
+}
+.menu__item {
+  width: 60px;
+  height: 40px;
+}
+.menu__list a {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0 auto;
+  width: 60px;
+  height: 40px;
+  color: #757575;
+  transition: all 0.5s ease;
+}
+.menu__list a:hover {
+  border-radius: 5px;
+  background-color: rgb(122, 138, 153);
+  text-align: center;
+  color: #fdfdfd;
+}
+.menu__list a:focus {
+  border: solid 1px rgb(122, 138, 153);
```

Comment:
You can rather use `outline` instead of `border`. 
It's better if it relates to the `:focus` state. I would like to say "you have to" :)

Differences Outline VS Border:
1. https://stackoverflow.com/questions/1158515/difference-between-outline-and-border#:~:text=Border%20is%20created%20inside%20the,outline%20draws%20outside%20the%20element.&text=Definition%20and%20Usage-,An%20outline%20is%20a%20line%20that%20is%20drawn%20around%20elements,the%20element%20%22stand%20out%22.

2. https://www.tutorialrepublic.com/css-tutorial/css-outline.php

Reply:
Thanks! Changed mistake.